documentation/modules/proc-deploying-api-server.adoc

```diff
@@ -7,46 +7,31 @@
 
 The API server provides a REST API for creating address spaces and addresses. It can also serve as an aggregated API server if it is registered as an API service.
 
-ifeval::["{PlatformName}" == "OpenShift"]
 .Prerequisites
-
+ifeval::["{PlatformName}" == "OpenShift"]
 * To install the {ProductName} API Server on {PlatformName}, you must have `cluster-admin` access to set up the
 required roles for delegating authentication to the {PlatformName} master; otherwise, you are restricted
 to using templates for creating address spaces. For more information about how to deploy without `cluster-admin` access,
 which restricts {ProductName} to using templates, see xref:installing-without-cluster-admin-privileges-{context}[].
 endif::[]
-
-.Procedure
-
-. Create a service account for the {ProductName} API server:
+* Modify the installation files according to the namespace the API Server is going to be installed in
```

Comment:
Modify the installation files to reflect the namespace in which the API Server will be installed: